nisdc/nisemidevicecontrol.py

```diff
@@ -1170,6 +1171,24 @@ def get_instrument_session(self, interface_name):
         except Exception as e:
             print("Exception occured at get instrument session")
             raise e
+        
+    def get_session_options(self, interface_name):
+        """Gets the grpc options for the instrument session.
+           Creates service server channel using the grpc options.
+
+            Return:
+                grpc_session_options { Object contains address {string}, port {int}, session_name{string} },
+                device_secure_channel { grpc.Channel }
```

Comment:
Can we create a class structure to hold the grpc channel and Session name?

Reply:
Updated